### DISCUSSION TITLE
Additional Tesla icons

### DISCUSSION THREAD

**Discussion by drjjr2** [elax46/custom-brand-icons#610] [Ideas]
I'm not cool enough to come up with SVG files or anything like that so what's the best way to get additional Tesla icons to match what their app has? I can get a screenshot from my iPhone but I don't know where to go from there. Difficulty: I'm no artist.

![53715](https://github.com/elax46/custom-brand-icons/assets/69729977/e6658928-67c6-407e-8a2d-1b215cae1ad1)

**Comment by elax46** [comment]
@rchiileea

**Comment by rchiileea** [comment]
put a request in @drjjr2 here https://github.com/elax46/custom-brand-icons/issues

**Reply by drjjr2** [reply]
Will do, thank you.
